test/sql/base.pg

```diff
@@ -619,6 +619,9 @@ SELECT is(table_version.ver_fix_revision_disorder(), 0::bigint,
 
 SELECT lives_ok('CREATE TABLE t (k int primary key, v text)');
 SELECT lives_ok($$ SELECT table_version.ver_enable_versioning('public','t') $$);
+SELECT lives_ok($$
+ ALTER TABLE table_version.public_t_revision DROP CONSTRAINT public_t_revision_check
+$$);
```

Comment:
This section of testing prepares some disordered revisions so that you have a test for the `ver_fix_revision_disorder()` function.. could it be adjusted slightly so that on first attempt at creating a disordered revision, you use `SELECT throws_like` to show the constraint protecting the disorder from occuring? And then run this `DROP CONSTRAINT` so that you can continue to actually create the disorder and then fix it.